plants/species/Sphenostylis/Sphenostylis_stenocarpa/Sphenostylis_stenocarpa_about.md

```diff
@@ -0,0 +1,16 @@
+**About *Sphenostylis stenocarpa***
+-------------------------
+*Sphenostylis stenocarpa* or the African yam bean is a species of 
+plant in the Fabaceae which is native to Africa. It is an important 
+source of food in many parts of Africa. The tubers are fried, boiled 
+or roasted, and are higher than the seeds in protein.
+
+
+Picture credit: [LICENSE TYPE]() via Wikimedia Commons [(Image source)]()
+
```

Comment:
```suggestion
Picture credit: [LICENSE TYPE]() via Wikimedia Commons [(Image source)]()
```
Has someone contacted the provider already to get an image?

Reply:
We didnt find an image in wiki and as per I understood its okay to not have an image but I will remove this and we can update it when we get an image